lib/registration.pm

```diff
@@ -214,6 +214,7 @@ sub register_addons {
             else {
                 assert_and_click("scc-code-field-$addon", timeout => 240);
             }
+            send_key 'alt-s';
```

Comment:
Actually, this key press should not be needed, as field is selected by `assert_and_click`. So we just need proper needle here. I guess the ones you've created just have too big area as click is performed in the center of it (AFAIK).

Reply:
I have modified the needle, and it is still failing to type the reg code

Reply:
* http://eris.suse.cz/tests/14632#step/addon_products_via_SCC_yast2/33

It's a bugger :/

Reply:
Thing is that name of the field depends on the name of the module, so we should at least use first letter of the `$addon` and then we might remove code above...

Reply:
```
[2019-05-21T12:31:38.732 CEST] [debug] <<< testapi::assert_and_click(mustmatch='scc-code-field-rt', button='timeout', timeout=240)
[2019-05-21T12:31:38.732 CEST] [debug] clicking at 502/495
```